### ISSUE TITLE
It does not delete files anymore

### ISSUE THREAD

**Issue by artuska** [diessica/slack-delete-files#12]
It shows it deleting but in fact it delete nothing. Maybe because Slack changed their API and this plugin just does not work because of it.

**Comment by diessica** [comment]
Hm.. I am using this: https://api.slack.com/methods/files.delete

Did you get any error message? I will have a look.

**Comment by artuska** [comment]
No, there are no error messages thrown, it just logs `{filename} has been deleted.` but in fact does not delete anything.

When I run `slack-delete-files --token={token}` it shows `Deleting 681 file(s)...` all the time even after task is finished and I run it again and again and again.

**Comment by diessica** [comment]
https://github.com/diessica/slack-delete-files/pull/10/files seems to fix it. I am going to release that as a new version.

Thanks for reporting.

**Comment by diessica** [comment]
v2.1.0 has been released! :tada:

Let me know if that solves your problem.

**Comment by artuska** [comment]
Now I get this:
> Error 'cant_delete_file' while deleting file.

**Comment by diessica** [comment]
Looking into Slack API, it seems like this error means "Authenticated user does not have permission to delete this file".

Does your token have the correct access permissions?

Reference:
![image](https://user-images.githubusercontent.com/5303585/45182093-a1ea8900-b220-11e8-9b67-9fe5e99a414e.png)

https://api.slack.com/methods/files.delete

**Comment by diessica** [comment]
Looking into Slack's API, it seems like this error means "Authenticated user does not have permission to delete this file".

Does your token have the correct access permissions?

Reference:
![image](https://user-images.githubusercontent.com/5303585/45182093-a1ea8900-b220-11e8-9b67-9fe5e99a414e.png)

https://api.slack.com/methods/files.delete

**Comment by artuska** [comment]
I've copied xoxp token from the Slack's API — https://api.slack.com/custom-integrations/legacy-tokens — I also clicked «Re-issue token» button but it affects nothing.

**Comment by dedmau5** [comment]
any news on this issue? I encountered the same with files actually seeming deleted within slack, but the storage seems unaffected. 
(my token is correct and I have the correct authority...)
